WooCommerce/Classes/ViewRelated/Dashboard/Settings/CardReadersV2/Tap to Pay Education/TapToPayEducationStepViewModel.swift

```diff
@@ -0,0 +1,30 @@
+import Foundation
+
+final class TapToPayEducationStepViewModel: ObservableObject {
```

Comment:
It might be good to start using `Observation` here, instead of Combine, depending when you plan to release. You could just wrap it all up in an availabilty check for iOS 17.

That said, without any Published properties, I don't think the `ObservableObject` conformance is needed, and you're not observing it anyway.

On the other hand, perhaps this could all just be held in properties on the View?

Reply:
> That said, without any Published properties, I don't think the ObservableObject conformance is needed, and you're not observing it anyway.

Good observation, true, that's a simple data structure.

> On the other hand, perhaps this could all just be held in properties on the View?

I still want to have some sort of data type to hold the data for the step. Maybe I will replace it with something else in the next tasks.